test/util/inEpsilon.go

```diff
@@ -0,0 +1,58 @@
+package alloratestutil
+
+import (
+	alloraMath "github.com/allora-network/allora-chain/math"
+	"github.com/stretchr/testify/require"
+)
+
+func (s *AlloraTestUtilSuite) inEpsilon(value alloraMath.Dec, target string, epsilon string, require *require.Assertions) {
```

Comment:
you could also pass `s.T` as a `t *testing.Testing` parameter then import the require package and use it as require.NoError(t, err)

I'm ok with something going in the setup that "sets up" the epsilon function, which I think is what this is

Reply:
frankly think the inEpsilon stuff can be its own pr, easier to parse that way,

let me see if I can push that out quickly